k8s/agones/chart/agones-mp/templates/installer.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: agones-installer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: "{{ .Release.Name }}"
+    helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  template:
+    spec:
+      serviceAccountName: "{{.Values.marketplace_integration.service_account}}"
+      containers:
+      - name: agones-installer
+        image: {{.Values.marketplace_integration.image}}
```

Comment:
The reason why I did not name this deployer is because my second phase  script is not the "-deployer" job. I wanted to avoid any confusion with that, and marketplace integration was actually a name that came from the experimental helm deployer.